address.go

```diff
@@ -44,6 +48,39 @@ func encodeAddress(hash160 []byte, netID byte) string {
 	return base58.CheckEncode(hash160[:ripemd160.Size], netID)
 }
 
+// encodeSegWitAddress creates a bech32 encoded address string representation
+// from witness version and witness program.
+func encodeSegWitAddress(hrp string, witnessVersion byte,
+	witnessProgram []byte) (string, error) {
+
+	// Group the address bytes into 5 bit groups, as this is what is used to
+	// encode each character in the address string.
+	converted, err := bech32.ConvertBits(witnessProgram, 8, 5, true)
+	if err != nil {
+		return "", err
+	}
+
+	// Concatenate the witness version and program, and encode the resulting bytes
```

Comment:
Within the project, lines are typically wrapped to 80 characters on a best-effort basis. This line in particular exceeds this limit (and a few others as well) as it's 86 characters long.